Sources/RichTextKit/Styles/RichTextStyleButton.swift

```diff
@@ -91,18 +91,23 @@ public extension RichTextStyleButton {
      */
     struct Style {
 
+        #if macOS
+        private let systemSpecificTextColor: Color? = Color(NSColor.textColor)
+        #else
+        private let systemSpecificTextColor: Color? = nil
+        #endif
         /**
          Create a rich text style button style.
-
+         
          - Parameters:
-           - inactiveColor: The color to apply when the button is inactive, by default `.primary`.
```

Comment:
Does this automatically happen when we run swift format? Because the params should have an inset.

I'm thinking of switching to /// which doesn't have this problem.

Reply:
I will make separate PR with swift format actually lot of stuff still not formatted.